docs/knowledgebase/parachains/launch.md

```diff
@@ -0,0 +1,316 @@
+staking (see Cumulus [implementation](https://github.com/paritytech/cumulus/blob/master/pallets/collator-selection/src/lib.rs)) or directly assigning collators via committee or other origins
+such as democracy.
+
+#### 2. Setup
+
+- Refer to [this instruction](https://substrate.dev/cumulus-workshop/#/en/3-parachains/1-launch?id=start-the-collator-node)
+to start and set up a collator node and [here](https://substrate.dev/cumulus-workshop/#/en/3-parachains/4-more-nodes?id=start-the-second-collator)
+to add more collator nodes.
+
+#### 3. Incentives
+
+- The [collator-selection pallet](https://github.com/paritytech/cumulus/blob/master/pallets/collator-selection/src/lib.rs)
+linked above already implements incentives via transaction fees. [This pallet](https://github.com/PureStake/moonbeam/blob/master/pallets/parachain-staking/src/lib.rs)
+implemented by Pure Stake team offers a more sophisticated scheme of parachain staking that supports inflationary monetary policy.
+
+#### 4. DevOps considerations
+
+- Ensure you follow [parachain DevOps best practices](https://gist.github.com/lovelaced/cddc1c7234b883ee37e71cf4a1d63cac).
+
+- We recommend you to watch the [recording](https://drive.google.com/file/d/1-nQ_SI2XK6vxPQvORWuv68Yj0UDz5FrO/view) of a dedicated DevOps for parachains office hour.
+
+### Runtime & Client Checklist
+
+#### 1. Unique Protocol ID
+
+There have been instances of networks having the same [Protocol ID](https://github.com/paritytech/substrate/blob/49a4103f4bfef55be20a5c6d26e18ff3003c3353/client/network/src/config.rs#L235) and different specs. **Ensure you customize your protocolID to make it universally unique**. Use a unique name & nonce/salt to ensure that this is not going to connect with another network.
+
+#### 2. Proper Weights & Benchmarking
+
+- Benchmark your runtime and use the generated weight functions (see our [benchmarking guide](../runtime/benchmarking)).
+Follow the benchmarking best practices. Don’t use default Substrate weights in production unless
+you know what you are doing.
+
+- We recommend having a block weight limit (block production time) of 0.5 seconds in the beginning
+due to uncertainties in block execution time. As the execution time of the network stabilizes the
+weights limit can be increased to 2 seconds.
+
+#### 3. Recommended Deployment of a Runtime
+
+> **NOTE**: Make sure to deploy the _compressed_ (not compact) version of your runtime to avoid
+> complications.
+
+- We advise you to launch your parachain being as slim as possible and do runtime upgrades to
+include functionality. The reason behind this is that when performing a runtime upgrade both the
+previous runtime and the new runtime are included in the PoV and therefore if the changes are too
+big it might not go through. **Always use the compressed version of the runtime to lower the amount
+of data being transferred.**
+
+  Still, in order to avoid storage upgrades, you could do the following:
```

Comment:
Is this going to be the default recommendation? This has been told to a few teams for now but is a hack and I don't think that is a good idea to make it a standard. 
Maybe @bkchr can give his PoV here.